### ISSUE TITLE
Link objects to encyclopedia articles

### ISSUE THREAD

**Issue by gjost** [densho/ddr-public#34]
DDR topics.json now has links from topic terms to related encyclopedia pages.

We would like to add lists of links to encyclopedia articles to these pages:
- DDR object detail page. http://ddr.densho.org/REPO/ORG/CID/EID/
- DDR topic leaf page. http://ddr.densho.org/browse/topics/NNN/

Lists will be built by
- listing topic terms attached to the `Entity`,
- compiling list of encyclopedia pages attached to those topic terms (from ddr repo or http://partner.densho.org/vocab/api/0.2/topics.json).
- getting article display titles from encyclopedia (see https://github.com/densho/encyc-front/issues/8).

Thoughts on the UI?

**Comment by gjost** [comment]
It's really not pretty yet (no article display titles) but got basic links working on topic term pages and on entity detail pages.

**Comment by gjost** [comment]
I coded up an API consumer that retrieved a URLs-to-titles `dict` from the encyclopedia (see densho/encyc-front#8), and it worked very nicely until I turned off my dev copy of `encyc-front` to work on something and `ddr-public` crashed.

The two systems should be able to connect 99.9% of the time, but to account for that 0.1% of the time we need to account for network timeouts and dropped connections.  To do this the Right Way would mean running Celery in the background, or a `cron` job or similar.

For now I decided to just fake it by running `urllib.unquote` on the URIs and removing the slashes.  This should work for most of the encyclopedia article titles.  If it doesn't we can try the API way.
